bubble_sort.rb

```diff
@@ -0,0 +1,49 @@
+# Internal: Swaps indexed element of an array with the next element.
+# Doesn't check if the elements exist. Modifies the array.
+#
+# %Array - Array to modify
+# %i - index of the element to be swapped
+#
+# Returns the modified array.
+def swap!(arr, idx)
+  temp = arr[idx]
+  arr[idx] = arr[idx + 1]
+  arr[idx + 1] = temp
```

Comment:
[Optional]

- [ ] Instead of using another variable what I will suggest is to use like that 👇.  It will make your code more nice and clean.

```
arr[i], arr[i+1] = arr[i+1], arr[i]
```